src/spyglass/decoding/v1/core.py

```diff
@@ -30,19 +31,23 @@ class DecodingParameters(SpyglassMixin, dj.Lookup):
 
     contents = [
         {
-            "decoding_param_name": "contfrag_clusterless",
+            "decoding_param_name": "contfrag_clusterless_"
```

Comment:
Edge case: if this package changes versioning systems to return something other than a string, this might error. Using an f-string might protect you f"contfrag_clusterless_{non_local_detector_version}", or you can reassign outside this `contents` scope. `nld_version_str = str(non_local_detector_version)` and then use `+` notation